moderation/reports/README.md

```diff
@@ -38,12 +38,20 @@ Should you need further information because something is unclear, or want to rep
 Make for each custom command file a separate custom command, preferrably in the same category to keep them neat and organized. Please make sure to follow each step precisely and to use the correct trigger and trigger type, as well.
 
 #### Here's what you have to do:
-1. Disable the native report command, found here: `Control Panel > Tools & Utilities > Moderation`
+1. Disable the native report command, found here: `Control Panel > Tools & Utilities > Moderation`, like [this](https://cdn.discordapp.com/attachments/767771719720632350/795330583303028746/unknown.png)
```

Comment:
Have you tried putting this as an actual image instead of linking it? Also: would be nice if we could change this to an imgur link or even a local asset 🤔

Reply:
This would require a rather huge readme update, as I would like to make every image an imgur link/local asset. I'll do these changes and call back later 👍